docs/reference/cli-reference/cli-ref-environment-variables.md

```diff
@@ -34,4 +34,6 @@ The list of properties documented in [NuGet pack and restore as MSBuild targets]
 | NUGET_UPDATEFILETIME_MAXRETRIES | Sets the number of times NuGet will attempt to set the file timestamp when extracting packages. | On Windows anti-virus software might temporarily open files, preventing NuGet from changing the timestamp. NuGet uses an exponential back-off where the wait duration between attempts is `Math.Pow(2, retryNumber)`. The default max retries is 9, meaning the default total wait duration before failure will be approximately one second. |
 | NUGET_XMLDOC_MODE | Determines how assemblies XML documentation file extraction should be handled. | Supported modes are *skip* (do not extract XML documentation files), *compress* (store XML doc files as a zip archive) or *none* (default, treat XML doc files as regular files). |
 | NUGET_CERT_REVOCATION_MODE | Determines how the revocation status check of the certificate used to sign a package, is performed when a signed package is installed or restored. When not set, defaults to `online`.| Possible values *online* (default), *offline*.  Related to [NU3028](../errors-and-warnings/NU3028.md) |
-
+| NUGET_ENABLE_ENHANCED_HTTP_RETRY | Enables or disables enhanced HTTP retry in NuGet. | Possible values are `true`  or `false` (default) |
```

Comment:
Do we want to keep this long term? Should we just not have it?

Reply:
Not long term but I'm documenting it since its now on by default and people might need to turn it off as an escape hatch for some unforseen reason.  Once we get rid of the old code, I'll remove this env var from the docs.